src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -66,6 +67,26 @@ public class SubscriptionConnection : IDisposable
         public long SubscriptionId { get; set; }
         public SubscriptionOpeningStrategy Strategy => _options.Strategy;
 
+        private ConcurrentQueue<string> _statusDescription = new ConcurrentQueue<string>();
+
+        public string[] GetRecentStatusDescriptions()
+        {
+            return _statusDescription?.ToArray()??Array.Empty<string>();
```

Comment:
when can this be null?

Reply:
After SubscriptionConnection being disposed